templates/partials/answers/address.html

```diff
@@ -23,9 +23,43 @@
         "label": _("Postcode"),
         "value": address_form.postcode._value() | e
     },
-    "uprn": {},
+    "uprn": {
+      "value": address_form.uprn._value() | e
+    },
     "searchButton": _("Search for an address"),
     "manualButton": _("Manually enter address")
-   } %}
+} %}
+
+{% if answer.lookup_options and address_lookup_api_url %}
+  {% set config = config | setAttribute("autosuggest", {
+    "label": {
+        "text": _("Enter address or postcode and select from results")
+    },
+    "isEditable": true,
+    "APIDomain": address_lookup_api_url,
+    "instructions": _("Use up and down keys to navigate suggestions once you’ve typed more than two characters. Use the enter key to select a suggestion. Touch device users, explore by touch or with swipe gestures."),
+    "ariaYouHaveSelected": _("You have selected"),
+    "ariaMinChars": _("Enter 3 or more characters for suggestions."),
+    "ariaOneResult": _("There is one suggestion available."),
+    "ariaNResults": _("There are {n} suggestions available."),
+    "ariaLimitedResults": _("Results have been limited to 10 suggestions. Type more characters to improve your search"),
+    "moreResults": _("Enter more of the address to improve results"),
+    "resultsTitle": _("Select an address"),
+    "noResults": _("No results found. Try entering a different part of the address"),
+    "tooManyResults": _("{n} results found. Enter more of the address to improve results"),
+    "typeMore": _("Enter more of the address to get results"),
+    "autocomplete": "new-password",
+    "errorTitle": _("There is a problem with your answer"),
+    "errorMessageEnter": _("Enter an address"),
+    "errorMessageSelect": _("Select an address"),
+    "errorMessageAPI": _("Sorry, there was a problem loading addresses"),
+    "errorMessageAPILinkText": _("Enter address manually"),
+    "options": {
+        "region_code": answer.lookup_options.region_code | lower,
+        "one_year_ago": "true" if answer.lookup_options.one_year_ago is defined and answer.lookup_options.one_year_ago else "false",
```

Comment:
How come these are these string values and not boolean?

Reply:
The macro expects strings (the values are assigned to html attributes).

Reply:
Ah ofcourse. 👍

Reply:
As discussed, this may need looking at in future as this doesn't match the doc. Hopefully in a future update, the macro itself can do the conversion.